src/DataBaseReadOnly.vala

```diff
@@ -759,6 +794,34 @@ public class FeedReader.DataBaseReadOnly : GLib.Object {
 		return tags;
 	}
 
+	private Gee.List<string> read_taggings(string articleID)
+	{
+		var list = new Gee.LinkedList<string>();
+
+		var rows = m_db.execute("SELECT * FROM taggings WHERE articleID = ?", { articleID });
+
+		foreach(var row in rows)
+		{
+			list.add(row[1].to_string());
+		}
+
+		return list;
+	}
+
+	private Gee.List<string> read_taggings_reverse(string tagID)
```

Comment:
Consider renaming these functions `read_taggings_by_article_id` and `read_taggings_by_tag_id`